File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-04/plagiarized/L4/05/pengulangan.kt
package IR

internal object inputoutput {
    fun hitHasil(m: Int) {
        var m = m
        while (m <= 10) {
            println(m.toString() + "\t\t" + m * 1.609)
            m++
            //ditambahin iteratornya
        }
    }

    @JvmStatic
    fun main(args: Array<String>) {
        val m = 1
        // print seusatu
        println("Miles\t\tKilometers")
        println("-------------------------------")
        // pake pengulangan while
        hitHasil(m)
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-04/plagiarized/L2/05/pengulangan.ts
import { println, printf } from 'util';

class InputOutput {
  public static main(args: string[]): void {
    println('Miles\t\tKilometers');
    println('-------------------------------');

    let m = 1;
    while (m <= 10) {
      printf('%d\t\t%.3f\n', m, m * 1.609);
      m++;
    }
  }
}

